src/main/resources/application.yml

```diff
@@ -9,6 +9,7 @@ app:
     admin-secret: 3%MXC<+fDbqdgVW
   security:
     jwtSecret: KtYxPmVMbnP%MgyG=53#=%3JjEQq!H4r
+    imdbApiKey: k_xqmbvnn2
```

Comment:
We can use this key for our app to make calls (this is my key, so don't abuse it if I need to use it :) ). The test data that is prepopulated should minimize use of this key until we really need to test the real API.